w2w/w2w.py

```diff
@@ -508,24 +500,87 @@ def wrf_remove_urban(
+# Get WRF grid info for Resampler
+# Inspired by: https://fabienmaussion.info/2018/01/06/wrf-projection/
+def _get_wrf_grid_info(info: Info) -> Dict[str, Dict[str, Any]]:
 
-    # Read  gridded WRF data
-    dst_data = xr.open_dataset(info.dst_nu_file)
+    # Initialize WGS84 projection
+    wgs_proj = pyproj.Proj(proj='latlong', datum='WGS84')
 
-    # Create simpler WRF grid target.
-    da_lu = xr.Dataset(
-        {'LU_INDEX': (['y', 'x'], dst_data['LU_INDEX'][0, :, :].values)},
-        coords={
-            'y': dst_data.XLAT_M.values[0, :, 0],
-            'x': dst_data.XLONG_M.values[0, 0, :],
-        },
-    )
+    # Read gridded WRF data
+    dst_data = xr.open_dataset(info.dst_file)
+
+    # Projection depends on value of MAP_PROJ
+    map_proj = int(dst_data.MAP_PROJ)
+
+    # Lambert Conformal Conic
+    if map_proj == 1:
+        wrf_proj = pyproj.Proj(
+            proj='lcc',
+            units='m',
+            a=6370000,
+            b=6370000,
+            lat_1=dst_data.TRUELAT1,
+            lat_2=dst_data.TRUELAT2,
+            lat_0=dst_data.MOAD_CEN_LAT,
+            lon_0=dst_data.STAND_LON,
+        )
+    # Polar Stereographic
+    elif map_proj == 2:
+        hemi = -90.0 if dst_data.TRUELAT1 < 0 else 90.0
+        wrf_proj = pyproj.Proj(
+            proj='stere',
+            units='m',
+            a=6370000,
+            b=6370000,
+            lat_0=hemi,
+            lon_0=dst_data.STAND_LON,
+            lat_ts=dst_data.TRUELAT1,
+        )
+    # Mercator
+    elif map_proj == 3:
+        wrf_proj = pyproj.Proj(
+            proj='merc',
+            units='m',
+            a=6370000,
+            b=6370000,
+            lon_0=dst_data.STAND_LON,
+            lat_ts=dst_data.TRUELAT1,
+        )
+    # Latlong - Equidistant Cylindrical
+    # Follow this: https://github.com/NCAR/wrf-python/blob/
+    # 4a9ff241c8f3615b6a5c94e10a945e8a39bdea27/src/wrf/projection.py#L928
+    elif map_proj == 6:
+        wrf_proj = pyproj.Proj(
+            proj='eqc',
+            units='m',
+            a=6370000,
+            b=6370000,
+            lon_0=dst_data.STAND_LON,
+        )
+
+    # Make transform
+    transformer_wrf = Transformer.from_proj(wgs_proj, wrf_proj)
+    e, n = transformer_wrf.transform(dst_data.CEN_LON, dst_data.CEN_LAT)
 
-    # Add projection information as attributes, save and read back in.
-    da_lu.rio.write_crs('epsg:4326', inplace=True)
-    da_lu.rio.to_raster(info.dst_gridinfo)
+    # Grid parameters
+    # https://github.com/fmaussion/salem/blob/
+    # d3f2e5e340c2af36c84c82a9de6099c90fba12e8/salem/wrftools.py#L734
+    dx, dy = dst_data.DX, dst_data.DY
+    nx, ny = dst_data.dims['west_east'], dst_data.dims['south_north']
+
+    # Down left corner of the domain
+    x0 = -(nx - 1) / 2.0 * dx + e
+    y0 = -(ny - 1) / 2.0 * dy + n
+
+    wrf_transform = Affine.translation(x0 - dx / 2, y0 - dy / 2) * Affine.scale(dx, dy)
+
+    wrf_grid_info: Dict[str, Dict[str, Any]] = {
```

Comment:
We don't need a type annotation here
```suggestion
    wrf_grid_info = {
```